### PR TITLE
Unexpected error 22 6 1

### PR DESCRIPTION
**Pull request by WawanSolihin**
The issue occured in RelateLevels function in Exporter.cs. In here, there is deletion of buildingstorey that is coincide causing some buildingstories to lose its data and later on triggers exception in Validation.

### CODE REVIEW

**Comment by WawanSolihin** [comment]
I will treat this as a draft change as I do not have a way to run regressions. This change has nothing to do with changes in 22.6.1. I am not sure that is not reported earlier. It should also occur in 22.6.0, but user said it works fine in 22.6.0. The code is just simply copied from the latest codes from 23.x.x for RelateLevels and other related functions

File: Source/Revit.IFC.Export/Exporter/Exporter.cs [hunk 1]
```diff
@@ -4199,29 +4199,140 @@ private void CreateGlobalCartesianOrigin(ExporterIFC exporterIFC)
          ExporterIFCUtils.SetGlobal2DOriginHandle(origin2d);
       }
 
-      private HashSet<IFCAnyHandle> RemoveContainedHandlesFromSet(ICollection<IFCAnyHandle> initialSet)
+      private static bool ValidateContainedHandle(IFCAnyHandle initialHandle)
```
Comment (AngelVelezSosa) [inline Source/Revit.IFC.Export/Exporter/Exporter.cs:4202]: @o-babii : don't we have code that keeps levels now?  This is a workaround but it seems better just not to delete the levels.

Comment (o-babii) [reply Source/Revit.IFC.Export/Exporter/Exporter.cs:4202]: Yes, we have, it wasn`t backsubmitted because of cpp changes.

**Review by o-babii — APPROVED** [review]
Ok, for me @WawanSolihin, 
I will rechek it after submitting to the Revit branch.
Thanks!

**Review by AngelVelezSosa — APPROVED** [review]

**Pull request merged**